front/src/app/api/tags/route.ts

```diff
@@ -0,0 +1,10 @@
+import { TagsData } from "@/services/prisma/TagsData";
+
+export const GET = async () => {
+    try {
+        const tags = await TagsData.getTags();
+        return Response.json({ tags });
+    } catch (err) {
+        return Response.json({ error: "failed to fetch tags from prisma" });
```

Comment:
Would it not also be helpful to return the error that was caught?

Reply:
True, new helper function added. See last commit.